doc/2/guides/main-concepts/3-permissions/index.md

```diff
@@ -235,16 +246,144 @@ Three different profiles can be created using that same role, each with varying
 }
 ```
 
+::: info
+Role restriction on indexes and collections can be used to **easily build multi-tenant application** were different groups of users cannot see each other data.
+:::
+
+### Rate limit
+
+<SinceBadge version="2.1.0" />
+
+The rate limit parameter controls how many API requests a user can send, **per second and per node**. Further requests made by a user that exceed the limit are rejected with a `429 Too Many Requests` error.
+
+::: info
+If no rate limit is defined, or if it is set to 0, then no limit is applied.
+If a user has several profiles with rate limits, the most permissive limit applies.
+:::
+
+::: warning
+Since unauthenticated users share the same user identifier, a rate limit set on the `anonymous` profile is applied to **all anonymous requests cumulated**, per second and per node. Except for the `auth:login` route, which is statically controlled in Kuzzle's configuration files.
+:::
+
+Example:
+
+```js
+{
+  "rateLimit": 20,
+  "policies": [ /* ...role policies, see below ... */ ]
+}
+```
+
+### Associated API actions
+
+ - [security:createOrReplaceProfile](/core/2/api/controllers/security/create-or-replace-profile/)
+ - [security:createProfile](/core/2/api/controllers/security/create-profile/)
+ - [security:deleteProfile](/core/2/api/controllers/security/delete-profile/)
+ - [security:getProfile](/core/2/api/controllers/security/get-profile/)
+ - [security:getProfileMapping](/core/2/api/controllers/security/get-profile-mapping/)
+ - [security:getProfileRights](/core/2/api/controllers/security/get-profile-rights/)
+ - [security:mDeleteProfiles](/core/2/api/controllers/security/m-delete-profiles/)
+ - [security:mGetProfiles](/core/2/api/controllers/security/m-get-profiles/)
+ - [security:scrollProfiles](/core/2/api/controllers/security/scroll-profiles/)
+ - [security:searchProfiles](/core/2/api/controllers/security/search-profiles/)
+ - [security:updateProfile](/core/2/api/controllers/security/update-profile/)
+ - [security:updateProfileMapping](/core/2/api/controllers/security/update-profile-mapping/)
+
 ---
 
+## Users
+
+Users can be assigned multiple profiles. These profiles will give them access to API actions via roles and their access will be limited by any restrictions defined in the profiles.
+
+Users are internal documents stored by Kuzzle. They contain two properties:
+ - `content`: profiles list and custom content
+ - `credentials`: available authentication credentials
```

Comment:
There are no credentials information stored in a user documents. I think that not even the list of known credentials for that user is stored. 
Only authentication plugins have that kind of information, there are strict separations between Kuzzle users and their credentials to prevent security issues.